extensions/sample-ui-plugin/src/client/publish/main.tsx

```diff
@@ -2,15 +2,16 @@
 // Licensed under the MIT License.
 
 import React, { useEffect, useCallback, useState } from 'react';
-import { setConfigIsValid, setPublishConfig, useConfigBeingEdited } from '@bfc/extension-client';
+import { closeDialog, onBack, savePublishConfig, useConfigBeingEdited } from '@bfc/extension-client';
 
-import { column, label, publishRoot, textField } from '../styles';
+import { backButton, buttonBar, column, label, publishRoot, saveButton, textField, title } from '../styles';
 
 export const Main: React.FC<{ title: string }> = (props) => {
   const [configBeingEdited] = useConfigBeingEdited();
   const [val1, setVal1] = useState(configBeingEdited ? configBeingEdited.val1 : '');
```

Comment:
val1,val2,val3 are not descriptive names. I do not understand what they are used for. Do give more meaningful variable names